docs/catalog/cloud/evaluate-whether-to-use-TLS-termination.md

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- security
+- compute
+- kubernetes
+- role:software-engineer
+- size:medium
+---
+
+# Evaluate whether to use TLS termination
+
+## Description
+Transport Layer Security (TLS) ensures that all data passed between the web server and web browsers remain private and encrypted. However, terminating and re-establishing TLS increases CPU utilization and might be unnecessary in certain architectures. 
+
+A balanced level of security can offer a more sustainable and energy efficient applications, while a higher level of security may increase the compute resource requirements.
+
+## Solution
+Consider if you can terminate TLS at your border gateway and continue with non-TLS to your application load balancer and onwards to your workload.
```

Comment:
It seems like this should be a consideration on the pattern and the solution should focus more on the action of terminating TLS at the border gateway.